tests/helper.t.ts

```diff
@@ -17,11 +17,12 @@
 const { readFileSync }: typeof import("fs") = require("fs");
 const { join }: typeof import("path") = require("path");
 
-export function JSONFromFile(file: string, folder: string) {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function JSONFromFile(file: string, folder: string): any {
```

Comment:
Should we consider `unknown` here?

Reply:
Could, looks to only be used in test code and would probably require changes in some other modules to handle the `unknown`